fe/fe-core/src/test/java/com/starrocks/pseudocluster/PseudoBackend.java

```diff
@@ -470,6 +475,28 @@ void handleTask(TAgentTaskRequest request) {
         }
     }
 
+    private void handleRealtimePush(TAgentTaskRequest request, TFinishTaskRequest finish) {
+        TPushReq pushReq = request.getPush_req();
+        TPushType pushType = pushReq.getPush_type();
+        switch (pushType) {
+            case DELETE:
+                handleRealtimePushTypeDelete(pushReq, finish);
+                break;
+            default:
+                throw new RuntimeException("pushType:" + pushType + " is not implement");
+        }
+    }
+
+    private void handleRealtimePushTypeDelete(TPushReq pushReq, TFinishTaskRequest finish) {
+        List<TTabletInfo> finish_tablet_infos = Lists.newArrayList();
+        long tabletId = pushReq.getTablet_id();
+        Tablet tablet = tabletManager.getTablet(tabletId);
+        finish_tablet_infos.add(tablet.getTabletInfo());
+        finish.setFinish_tablet_infos(finish_tablet_infos);
+        finish.setRequest_version(pushReq.getVersion());
+    }
```

Comment:
when should push increase tablet's version? in publish stage? Is a rowset required?

Reply:
In fact, there is no logic for increase versions, nor logic for logical deletion according to conditions.
Does this need to add logic to increase the version number?